adsmsg/tests/test_nonbibcode_record.py

```diff
@@ -14,13 +16,37 @@ def tearDown(self):
     def test(self):
         nonbib_data = {'bibcode': '2003ASPC..295..361M', 'refereed': False,
                        'downloads': [0,0], 'boost': 3.1,
-                       'reads': [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]}
+                       'reads': [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20],
+                       'data_links': {'a': ['b', 'c'], 'w': ['x', 'y', 'z']},
```

Comment:
i'm confused: what is using/needing 'data_links'? solr consumes 'data' (and in this unittest, it would read 'a:1', and 'w:3'), data_links_rows is for the resolver